### PR TITLE
:memo: add talks (web2day / bdx.io)

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+  </a>
+</p>
 
-## Sources
- - [reactive programming -medium- @tkssharma](https://medium.com/@tkssharma/reactive-programming-rx-js-introduction-32bf963eee1b)
- - [xstream README.md](https://github.com/staltz/xstream)
- - [cycle.js.org documentation](https://cycle.js.org/getting-started.html)
+# Talks
+## Who
+ - alakarte [[Twitter](https://twitter.com/alakarteio) | [Github](https://github.com/alakarteio)]
```
Comment (fabienjuif) [inline README.md]: This is not the right place

**Comment by fabienjuif** [comment]
this is not `:readme:` but `:memo:`

**Review by fabienjuif — CHANGES_REQUESTED** [review]
comments

File: README.md (hunk 1 above)
Comment (guillaumecrespel) [reply README.md]: :ok_hand:

**Review by fabienjuif — APPROVED** [review]

**Pull request merged**